crates/cxx-qt-lib-extras/include/qml/qjsengine.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <memory>
+
+#include <QtQml/QJSEngine>
+
+#include "rust/cxx.h"
+
+namespace rust
+{
+    namespace cxxqtlib2
+    {
+        ::std::unique_ptr<QJSEngine>
+        qjsengineNew();
```

Comment:
It should be possible to replace this with our make_unique template from cxx-qt-lib.

Take a look at the `qpainter_init_default()` function in qpainter.rs for how to use this.